src/Common/src/System/IO/DelegatingStream.cs

```diff
@@ -172,6 +172,11 @@ public override void EndWrite(IAsyncResult asyncResult)
             _innerStream.EndWrite(asyncResult);
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
+        {
+            _innerStream.CopyTo(destination, bufferSize);
+        }
```

Comment:
@stephentoub 

> Do we ever actually expose this stream anywhere where CopyTo might be called on it?

Yeah, we do. For example, `StreamContent.CreateContentReadStreamAsync` [returns a `ReadOnlyStream`](https://github.com/dotnet/corefx/blob/c351ba1f094871e4ae97f23f20f26648662cc4e6/src/System.Net.Http/src/System/Net/Http/StreamContent.cs) which inherits from this class, and the result of that can be passed out of `HttpContent.ReadAsStreamAsync` which is public.

What's the moniker I should put after #if here?